sdlib/d/gc/sizeclass.d

```diff
@@ -176,6 +176,10 @@ auto getBinInfos() {
 	return bins;
 }
 
+bool isSmall(ubyte sizeClass) {
+	return sizeClass < ClassCount.Small;
+}
```

Comment:
ok, this looks like a possibly useful refactoring. Let's focus on that guy.
1. Move it near the data it uses (in this case, `ClassCount`).
2. Give it an explicit name. `isSmall` does not indicate whether it expects a size class or a size, or the age of the captain.
3. Convert the codebase toward using this. This operation is done all over the place, if we are going to use a utility function to do this check, then let's do this, instead of creating a mixture of different things.
4. Do we need a corresponding check for large and huge sizes?

Reply:
I don't think 4 is needed right now.
The rest is in now.
Also added the routine which uses the size class estimator mechanism and tests for same.